pubspec.yaml

```diff
@@ -5,9 +5,10 @@ description: Utilities for diffing two JSON objects
 homepage: https://github.com/google/dart-json_diff
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.8.4 <3.0.0'
 dependencies:
-  collection: ^1.14.11
+  collection: ^1.14.13
   diff_match_patch: ^0.3.0
 dev_dependencies:
-  test: ^1.6.4
+  test: ^1.15.3
```

Comment:
Why does this code require a newer version of test?

Reply:
That was me making a bit broad update strokes. Probably not necessary.